pkg/js/tests/validate-rules.test.ts

```diff
@@ -0,0 +1,70 @@
+import { describe, expect, it } from "@jest/globals";
+import { Validate } from "../validator/validate-rules";
+
+describe("validation rules", () => {
+  describe("Rule 'Object'", () => {
+    it("should pass '<object>:<id>'", () => {
+      expect(Validate.object("document:1")).toBeTruthy();
+    });
+
+    it("should fail if no ':' delimiter", () => {
```

Comment:
Can you add more test cases?

These should fail
* document::1
* :document:1
* document:1:
* document#1
* document:1#relation

Reply:
Separating test cases into 2 groups, those specific to a validator, and those which are just checks for bad formats that should universally fail.